fintorch/models/timeseries/tft/tft_module.py

```
@@ -0,0 +1,206 @@
+        number_of_past_inputs,
+        horizon,
+        embedding_size_inputs,
+        hidden_dimension,
+        dropout,
+        number_of_heads,
+        past_inputs,
+        future_inputs,
+        static_inputs,
+        batch_size,
+        device,
+        quantiles=[0.1, 0.5, 0.9],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)